### DISCUSSION TITLE
@vinxi/solid vs @solidjs/start

### DISCUSSION THREAD

**Discussion by RonenEizen** [solidjs/solid-start#1386] [Q&A] [answered]
I am looking at vinxi's solidjs examples and I see that they use @vinxi/solid. How is that package different from @solidjs/start?

**Comment by ryansolid** [comment, marked as answer]
It was Nikhil's initial stab to show me how he'd implement it. He wanted to prove it could do everything our other solutions could. He basically hasn't touched that in 9 months. And SolidStart continued to implement more things.
